modules/simul/src/main/SimulForm.scala

```diff
@@ -108,6 +111,7 @@ object SimulForm {
         "position" -> optional(lila.common.Form.fen.playableStrict),
         "color"    -> stringIn(colorChoices),
         "text"     -> cleanText,
+        "estimatedStartAt" -> optional(inTheFuture(ISODateTimeOrTimestamp.isoDateTimeOrTimestamp)),
```

Comment:
doesn't this prevent updating the simul after it started? To update its description for instance.

Reply:
Good point.

A couple of ways I could adress this,
1. I remove the inTheFuture validation entirely, or
2. I update the "baseForm"-method, which is used for both create and edit, to make use of the knowledge if this is a "create" or a "edit" form, and only apply inTheFuture validation on the "create" form

It seems it is not (currently) possible to update a simul after it has started,
at least not from the "usual" cogwheel option.
Screenshots of non-started + started simul,
![simul_update](https://user-images.githubusercontent.com/4084220/118355310-9625f780-b56f-11eb-8de4-27cf806317fd.png)

but it sounds reasonable that it should be possible to update any simul, running or not,
and having the "inTheFuture"-validation on this field might cause unexpected harm.